src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js

```diff
@@ -0,0 +1,68 @@
+export const uiSchema = {
+  'ui:title': 'Certification',
+  unemployability: {
+    'view:statementsAreTrue': {
+      'ui:title':
+        'I certify that because of my service-connected disabilities, I’m unable to hold down a steady job and that the statements in this claim are true and complete to the best of my knowledge and belief. I understand that these statements will be considered in deciding my eligibility for Individual Unemployability benefits.',
+      'view:statementsAreTrueAccept': {
+        'ui:title': 'I accept',
+      },
+      'ui:options': {
+        showFieldLabel: true,
+      },
+      'ui:validations': [
+        (errors, item) => {
+          if (!item['view:statementsAreTrueAccept']) {
+            errors.addError('You must accept the certification');
+          }
+        },
+      ],
+    },
+    'view:informOfReturnToWork': {
+      'ui:title':
+        'I understand that if I become eligible for Individual Unemployability benefits, I must immediately inform VA if I return to work. I also understand that total disability benefits paid to me after I begin work may be considered an overpayment, and I may be asked to repay money to VA.',
+      'view:informOfReturnToWorkAccept': {
+        'ui:title': 'I accept',
+      },
+      'ui:options': {
+        showFieldLabel: true,
+      },
+      'ui:validations': [
+        (errors, item) => {
+          if (!item['view:informOfReturnToWorkAccept']) {
+            errors.addError('You must accept the certification');
+          }
+        },
+      ],
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    unemployability: {
```

Comment:
I'm wondering if we should follow this method throughout all of 8940 questions (at least the ones related to the schema). Maybe bring this up with the team so we're all on the same page.